src/posts/hawaii-hackathon-2018.mdx

```diff
@@ -9,7 +9,7 @@ pubDate: '2019-05-13T07:51:37+00:00'
 layout: '../layouts/BlogPost.astro'
 id: 1316
 guid: 'https://groupincome.org/?p=1316'
-permalink: /2019/05/group-income-update-hawaii-hackathon-2018/
+permalink: /posts/group-income-update-hawaii-hackathon-2018/
```

Comment:
These URLs cannot be changed, because doing so will break any links on other websites to our old blog posts.

Is it possible to keep the original URLs?

Reply:
@taoeffect Yes, I can make the updates again then.